postinstall script failed

```
> react-jsx-parser@1.28.2 postinstall /app/node_modules/react-jsx-parser
> yarn patch-package

yarn run v1.17.3
[91merror Command "patch-package" not found.
[0minfo Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
[91mnpm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.2 (node_modules/react-scripts/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
[0m[91mnpm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.2.7 (node_modules/jest-haste-map/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.3.1 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.2.7 (node_modules/watchpack-chokidar2/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.2.7 (node_modules/webpack-dev-server/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
[0m[91mnpm WARN bootstrap@4.6.0 requires a peer of jquery@1.9.1 - 3 but none is installed. You must install peer dependencies yourself.
npm WARN connected-react-router@6.8.0 requires a peer of immutable@^3.8.1 || ^4.0.0-rc.1 but none is installed. You must install peer dependencies yourself.
npm WARN connected-react-router@6.8.0 requires a peer of seamless-immutable@^7.1.3 but none is installed. You must install peer dependencies yourself.
npm WARN react-scripts@3.4.4 requires a peer of typescript@^3.2.1 but none is installed. You must install peer dependencies yourself.
npm WARN react-use-dimensions@1.2.1 requires a peer of typescript@^3.5.2 but none is installed. You must install peer dependencies yourself.
npm WARN simple-react-lightbox@3.6.3 requires a peer of react@^17.0.1 but none is installed. You must install peer dependencies yourself.
npm WARN simple-react-lightbox@3.6.3 requires a peer of react-dom@^17.0.1 but none is installed. You must install peer dependencies yourself.
npm WARN react-jsx-parser@1.28.2 requires a peer of react@>=17 but none is installed. You must install peer dependencies yourself.
npm WARN react-jsx-parser@1.28.2 requires a peer of react-dom@>=17 but none is installed. You must install peer dependencies yourself.
npm WARN sass-loader@8.0.2 requires a peer of sass@^1.3.0 but none is installed. You must install peer dependencies yourself.
npm WARN sass-loader@8.0.2 requires a peer of fibers@>= 3.1.0 but none is installed. You must install peer dependencies yourself.
npm WARN rollup-plugin-typescript2@0.25.3 requires a peer of rollup@>=1.26.3 but none is installed. You must install peer dependencies yourself.
npm WARN @pmmmwh/react-refresh-webpack-plugin@0.3.3 requires a peer of @types/webpack@^4.41.12 but none is installed. You must install peer dependencies yourself.
npm WARN @pmmmwh/react-refresh-webpack-plugin@0.3.3 requires a peer of type-fest@^0.13.1 but none is installed. You must install peer dependencies yourself.
npm WARN @pmmmwh/react-refresh-webpack-plugin@0.3.3 requires a peer of webpack-hot-middleware@2.x but none is installed. You must install peer dependencies yourself.
npm WARN @pmmmwh/react-refresh-webpack-plugin@0.3.3 requires a peer of webpack-plugin-serve@0.x || 1.x but none is installed. You must install peer dependencies yourself.
[0m[91m
[0m[91mnpm ERR! code[0m[91m ELIFECYCLE
npm ERR! errno 1
[0m[91mnpm ERR! react-jsx-parser@1.28.2 postinstall: `yarn patch-package`
npm ERR! Exit status 1
[0m[91mnpm ERR! 
npm ERR! Failed at the react-jsx-parser@1.28.2 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
[0m[91m
npm ERR![0m[91m A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-02-08T10_45_59_479Z-debug.log
[0mThe command '/bin/sh -c npm install' returned a non-zero code: 1
```

Probably because of replacing this `patch-package` package - https://github.com/TroyAlford/react-jsx-parser/commit/bec2ef09a784978d0085f4a43c18151e8b8deaed#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R54
But you still calling it in your postinstall script `"postinstall": "yarn patch-package",`

--- a/package.json
+++ b/package.json
@@ -14,7 +14,7 @@
   "types": "dist/index.d.ts",
   "name": "react-jsx-parser",
   "repository": "TroyAlford/react-jsx-parser",
-  "version": "1.28.2",
+  "version": "1.28.3",
   "dependencies": {
     "@types/jsdom": "^16.2.6",
     "acorn": "^8.0.5",

--- a/package.json
+++ b/package.json
@@ -19,8 +19,7 @@
     "@types/jsdom": "^16.2.6",
     "acorn": "^8.0.5",
     "acorn-jsx": "^5.3.1",
-    "core-js": "^3.8.3",
-    "patch-package": "^6.2.2"
+    "core-js": "^3.8.3"
   },
   "devDependencies": {
     "@babel/core": "^7.12.13",
@@ -77,9 +76,9 @@
     "merge": "^1.2.1"
   },
   "scripts": {
-    "build": "yarn patch-package && yarn types && cross-env NODE_ENV=production webpack",
+    "build": "npx patch-package && yarn types && cross-env NODE_ENV=production webpack",
     "develop": "NODE_ENV=production concurrently -n build,ts,demo -c green,cyan,yellow \"yarn webpack --watch\" \"yarn types --watch\" \"yarn webpack serve --config ./webpack.demo.js\"",
-    "postinstall": "yarn patch-package",
+    "postinstall": "npx patch-package",
     "prebuild": "mkdirp ./dist && rm -rf ./dist/*",
     "sourcemap": "yarn build && source-map-explorer ./dist/es5/react-jsx-parser.min.js",
     "test": "jest",

